template.rc

```diff
@@ -0,0 +1,57 @@
+/* This is a template RC file.
+ * Do not edit with MSVC */
+#ifdef APSTUDIO_INVOKED
+# error dont edit with MSVC
+#endif
+
+#include "winres.h"
+#include "main/php_version.h"
+#include "phongo_version.h"
+
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
+#pragma code_page(1252)
+
+#ifndef THANKS_GUYS
+# define THANKS_GUYS ""
+#endif
+
+//Version
+VS_VERSION_INFO VERSIONINFO
+ FILEVERSION PHP_MONGODB_VERSION_DESC
+ PRODUCTVERSION PHP_MONGODB_VERSION_DESC
+ FILEFLAGSMASK 0x3fL
+#ifdef _DEBUG
+ FILEFLAGS VS_FF_DEBUG
+#else
+ FILEFLAGS 0x0L
+#endif
+ FILEOS VOS__WINDOWS32
+ FILETYPE VFT_DLL
+ FILESUBTYPE VFT2_UNKNOWN
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904b0"
+        BEGIN
+            VALUE "Comments", THANKS_GUYS "\0"
+            VALUE "CompanyName", "MongoDB, Inc.\0"
+            VALUE "FileDescription", FILE_DESCRIPTION "\0"
+            VALUE "FileVersion", PHP_MONGODB_VERSION
```

Comment:
Is this being pulled directly from `phongo_version.h`? If so, both it and `THANKS_GUYS` are declared with `#define`. Why do you add a `"\0"` to `THANKS_GUYS` but not here?

Reply:
Don't know. I copied this from Xdebug where it worked for the past 10 years :-)